xbmc/addons/PVRClient.cpp

```diff
@@ -365,7 +365,7 @@ void CPVRClient::WriteClientChannelInfo(const CPVRChannelPtr &xbmcChannel, PVR_C
 bool CPVRClient::GetAddonProperties(void)
 {
   std::string strBackendName, strConnectionString, strFriendlyName, strBackendVersion, strBackendHostname;
-  PVR_ADDON_CAPABILITIES addonCapabilities = {0};
+  PVR_ADDON_CAPABILITIES addonCapabilities = {false};
```

Comment:
`PVR_ADDON_CAPABILITIES` struct has members which are not of type bool.

Reply:
Yep, please reverse this change.